src/common.props

```diff
@@ -3,7 +3,7 @@
   <!-- Version settings: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/ -->
   <PropertyGroup>
 	<MajorVersion>2</MajorVersion>
-	<MinorVersion>1</MinorVersion>
```

Comment:
Don't we need to do a major release here? 

From my understanding (and I think @davidmrdavid mentioned this before) we cannot make a major version update of a dependency without also doing major version update of this package.

Reply:
I agree. In-proc customers will have their DTF.Core (and DTFx.AS, due to the fact that the WebJobs extension imports the entire DTFx.AS backend _even when using Netherite_) version majorly upgraded.

Therefore, this should be `3.0.0`.